batch/batch/driver/instance.py

```diff
@@ -159,7 +159,7 @@ async def activate(self, ip_address, timestamp):
         assert self._state == 'pending'
 
         rv = await check_call_procedure(
-            self.db, 'CALL activate_instance(%s, %s, %s);', (self.name, ip_address, timestamp)
+            self.db, 'CALL activate_instance(%s, %s, %s);', (self.name, ip_address, timestamp), 'activate'
```

Comment:
Let's make these names match the SQL procedure that we're CALLing here (`activate_instance`) and below with deactivate as well.